### PR TITLE
Admin views tests

### PR DESCRIPTION
**Pull request by somexpert**
Tests for the admin views as well as the lone helper function within admin_views.py.

### CODE REVIEW

**Comment by somexpert** [comment]
@ldko, @damonkelley, this is ready for review.

File: tests/test_admin_views.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+        response = admin_views.duplicates(request, 'donor')
+
+        assert response.status_code == 200
+
+    def test_returns_200_with_unmapped_model(self, rf, admin_user):
+        request = rf.get('/admin/accession/donor/duplicates/')
+        request.user = admin_user
+
+        response = admin_views.duplicates(request, 'donor')
```
Comment (ldko) [inline tests/test_admin_views.py]: 'donor' is mapped. Same as above test.

Comment (somexpert) [inline tests/test_admin_views.py]: Ah, right, did some modifying and forgot to fix that one. On it.

File: tests/test_admin_views.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+        request = rf.get('/admin/accession/accession/1/print/')
+        request.user = admin_user
+
+        response = admin_views.print_view(request, 'accession', 'accession', 1)
+
+        # Check a couple fields to make sure they're there.
+        assert accession.description in response.content
+        assert accession.acquisition_method in response.content
+        # Make sure the related items is there too.
```
Comment (ldko) [inline tests/test_admin_views.py]: Change to "related items are"? Besides that, :+1: .

File: tests/test_admin_views.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+    def test_returns_200_with_unmapped_model(self, rf, admin_user):
+        request = rf.get('/admin/accession/city/duplicates/')
+        request.user = admin_user
+
+        response = admin_views.duplicates(request, 'city')
+
+        assert response.status_code == 200
+
+    @pytest.mark.xfail(reason='The view does not catch the exception raised'
```
Comment (damonkelley) [inline tests/test_admin_views.py:30]: Are we expecting an `HttpResponseNotFound` or an `Http404` exception here?

**Comment by damonkelley** [comment]
So after thinking about it a bit, I think we should remove the URIs from request factory methods. I like that it gives the reader an idea of what the endpoint that the view sits behind looks like, but it is information that the views don't use, so it could end up being misleading.

**Comment by somexpert** [comment]
Alright, I'll remove them. I suppose it also could be a problem if those views were to change. If we weren't careful, those URIs might not be updated since they aren't required for the tests to work, and would only cause confusion.

File: tests/test_admin_views.py (hunk 3 above)
Comment (somexpert) [inline tests/test_admin_views.py:30]: Hmmmm, I'm beginning to think that this test should actually be a test for the query_to_tuple() function. What do you think?

Comment (damonkelley) [inline tests/test_admin_views.py:30]: it is probably fine for now

**Comment by damonkelley** [comment]
:+1:

File: tests/test_admin_views.py [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+from accession import admin_views
+
+from .factories import DonorFactory, AccessionFactory
+
+pytestmark = pytest.mark.django_db
+
+
+class TestDuplicates():
+
+    def test_returns_200_with_mapped_model(self, rf, admin_user):
```
Comment (damonkelley) [inline tests/test_admin_views.py]: What do you mean by _mapped_? I see there is a `field_mapping` but this is unrelated to the `Donor` model.

**Comment by damonkelley** [comment]
:+1:
